app/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
     buildToolsVersion '26.0.2'
     defaultConfig {
         applicationId "io.appium.uiautomator2"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
is this a change that will affect users?

Reply:
We do check this in the driver code as well and throw an error if the API version is below 21